src/common/models/essence/essence.ts

```diff
@@ -190,8 +190,6 @@ export class Essence extends ImmutableRecord<EssenceValue>(defaultEssence) {
     return timeFilter;
   }
 
-  public visResolve: Resolve;
```

Comment:
This cause to `visResolve` to be a `undefined` during runtime.

Reply:
So probably we will encounter bunch of different weird issues.